packages/blocks/src/api/raw-handling/html-to-blocks.js

```diff
@@ -19,7 +19,7 @@ export function htmlToBlocks( html ) {
 
 	doc.body.innerHTML = html;
 
-	return Array.from( doc.body.children ).map( ( node ) => {
+	return Array.from( doc.body.children ).flatMap( ( node ) => {
```

Comment:
👋 I tested this changes in the native version and unfortunately the built-in version of `flatpMap` is not 100% supported (Android particularly).

Would it be possible to use the version from the `lodash` library instead? Do you foresee any inconvenience of using that version instead the built-in?

Thanks!

Reply:
Thanks for checking! 🙏

Using the lodash one instead should be fine, but I'm wondering whether our Babel config is intentionally leaving it unhandled. Could it have been an oversight in the [v7 migration](https://babeljs.io/docs/en/v7-migration#remove-proposal-polyfills-in-babelpolyfillhttpsgithubcombabelbabelissues8416)?

I'd think it would be safer if we relied on Babel to handle it, unless there's a project-wide preference to use lodash replacements for built-in functions. I'm not really familiar with the goings-on here though. Do you know?

Reply:
(Noting one more instance, in case we're going the lodash route.)

https://github.com/WordPress/gutenberg/blob/004a32ec09d5b789be694d85725fe49cca385f08/packages/edit-navigation/src/store/utils.js#L136

Reply:
> Using the lodash one instead should be fine, but I'm wondering whether our Babel config is intentionally leaving it unhandled. Could it have been an oversight in the v7 migration?

To be honest I'm not sure, the adoption of built-in `flatMap` is quite high (`91.18%` as checked in [caniuse.com](https://caniuse.com/mdn-javascript_builtins_array_flatmap)) and since it has been only used in one place maybe we haven't spotted this issue yet.

> (Noting one more instance, in case we're going the lodash route.)

Good catch! That part should be also incompatible with the native version but most likely it hasn't raised any issue because is not used (I need to check it).

I found interesting that this place was the only one where the built-in `flatMap` was being used so far, maybe as you commented, there's a project-wide preference to use `lodash`.
I'm not familiar neither with this since I'm working more in the native side but I'll ask for help.

Reply:
The poly fills are included separate in WordPress in a `wp-polyfill` package. I think that package is probably not included in native, should it be included?

Reply:
Yeah we would need to import and inject that package when generating the JS bundle for React native. Could you point me out where I can get that package? Thanks!

Reply:
Its setup is a bit complex, it's mainly composed of `babel-polypill` see https://github.com/WordPress/wordpress-develop/blob/master/package.json#L81

but it has other dependencies that are only loaded if the browser doesn't support them https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/script-loader.php#L122-L136

Reply:
Thanks @youknowriad for the info!

I'm going to try adding the `core-js` dependency to the `react-native-editor` package (the entry point for the native version) and import the polyfill for `flatMap` from `core-js/features/array/flat-map` as states in the [Babel documentation](https://babeljs.io/docs/en/v7-migration#remove-proposal-polyfills-in-babelpolyfillhttpsgithubcombabelbabelissues8416).

Reply:
Finally I've included the polyfill by adding `core-js` to the native package in this [PR](https://github.com/WordPress/gutenberg/pull/28400). In the future if we need to add new polyfills we'll have an entry point there.

Thanks @mirka and @youknowriad for checking this!

Reply:
Thanks for the quick fix @fluiddot 🙌